ProcessLib/HT/HTProcess.cpp

```diff
@@ -160,5 +180,54 @@ void HTProcess::setCoupledTermForTheStaggeredSchemeToLocalAssemblers()
         _local_assemblers, _coupled_solutions);
 }
 
+std::tuple<NumLib::LocalToGlobalIndexMap*, bool>
+    HTProcess::getDOFTableForExtrapolatorData() const
```

Comment:
It can be common to  all processes by specify one DOF table for extrapolation. To do this we have to  replace `Process::_local_to_global_index_map` with  `Process::_local_to_global_index_map_single_component` 
I prefer to  try it later in other PR.

Reply:
OK, later PR is totally fine and it's not too important.